ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -345,177 +144,58 @@ public void close() throws IOException {
       shutDownExecutorService(scheduler, config.updaterShutDownTimeoutMinutes, TimeUnit.MINUTES);
     }
     maybeUnsubscribeChangeTopic();
-    open.set(false);
+    cachedAccountService.close();
```

Comment:
I think we might need to move `maybeUnsubscribeChangeTopic` from `MySqlAccountService.close` to `CachedAccountService.close` since the topics are subscribed from `CachedAccountService`.

Reply:
Sure, removed.